test/unit/models/publishes_to_publishing_api_test.rb

```diff
@@ -47,4 +47,15 @@ class PublishesToPublishingApiTest < ActiveSupport::TestCase
     Whitehall::PublishingApi.expects(:publish_async).never
     organisation.update_attribute(:name, 'Edited org')
   end
+
+  test "update publishes to Publishing API using :en locale when no translated fields are set" do
+    person = create(:person, attributes_for(:person).except(:biography))
+
+    content_item = PublishingApiPresenters.presenter_for(person).as_json
+    expected_publish_request = stub_publishing_api_put_item(person.search_link, content_item)
+
+    person.update_attribute(:forename, 'Edited person')
+
+    assert_requested expected_publish_request
```

Comment:
I can't see where we're actually testing that the locale is sent here, especially as the implementation appears to be slightly wrong - is there something I'm missing?